src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/config_zh_CN.properties

```diff
@@ -36,7 +36,7 @@ Concurrency\ Limit=\u5BB9\u5668\u6570\u91CF
 Apply\ cap\ only\ on\ alive\ pods=\u7528\u4E8E\u6D3B\u8DC3\u7684 Pod
 Max\ connections\ to\ Kubernetes\ API=\u8FDE\u63A5 Kubernetes API \u7684\u6700\u5927\u8FDE\u63A5\u6570
 Container\ Cleanup\ Timeout\ (minutes)=\u5BB9\u5668\u8D85\u65F6\uFF08\u5206\uFF09
-Transfer\ proxy\ related\ environment\ variables\ from\ master\ to\ agent=\u4ECE master \u4F20\u9012\u7ED9 agent \u7684\u73AF\u5883\u53D8\u91CF
+Transfer\ proxy\ related\ environment\ variables\ from\ controller\ to\ agent=\u4ECE controller \u4F20\u9012\u7ED9 agent \u7684\u73AF\u5883\u53D8\u91CF
```

Comment:
The left text has been changed in Jelly, we need to change it here too to have translation working.